src/averageAges.js

```diff
@@ -20,6 +20,19 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let summaryAge = 0;
+  let result = 0;
+
+  const men = (century) ? people.filter(person => {
+    return person.sex === 'm' && Math.ceil(person.died / 100) === century;
+  }) : people.filter(person => person.sex === 'm');
+
+  summaryAge = men
+    .reduce((prev, current) => prev + current.died - current.born, 0);
+
+  result = summaryAge / men.length;
```

Comment:
you don't reassign these variables, should they be defined at the start of the function?